src/test/App.test.js

```diff
@@ -36,7 +39,19 @@ describe("App", () => {
     props = playerTurn.props();
     expect(props.player).toBe(PLAYER.X);
   });
-  it("can determine X as winner", () => {});
+  it("can determine X as winner", () => {
```

Comment:
Does this work with the updates or not yet?

Reply:
It  does not.

Reply:
ok